### ISSUE TITLE
Can LayerDiffusion be executed after ADetailer?

### ISSUE THREAD

**Issue by Shinnri-Liu** [layerdiffusion/sd-forge-layerdiffusion#12]

**Comment by deeplearner5** [comment]
+1 - this would be really useful, if viable. I always use ADetailer to enhance facial detailing.

**Comment by MNeMoNiCuZ** [comment]
Agreed. If the order of execution could be tweaked, not sure if it's up to each extension or maybe that's a feature request for A1111/Forge?

**Comment by newxhy** [comment]
ControlNet Integrated It's also necessary

**Comment by git-uniqity** [comment]
so comfyui edition is necessary, for the adhustable workflow

**Comment by LoveHeiZhu** [comment]
Having the same problem, looking forward to an update fix or other setup method

**Comment by new-sankaku** [comment]
There may be a fix, but I forked it and fixed it briefly.
Hires, I think ADtailer will work.
Please try using it if you like.

https://github.com/new-sankaku/sd-forge-layerdiffuse.git

Note: Does Hires cause an error if the image size after enlargement is not a multiple of 256? We recommend multiplying 1024*1024 by 1.5 to 1536x1536.

**Comment by deeplearner5** [comment]
> There may be a fix, but I forked it and fixed it briefly. Hires, I think ADtailer will work. Please try using it if you like.
> 
> https://github.com/new-sankaku/sd-forge-layerdiffuse.git
> 
> Note: Does Hires cause an error if the image size after enlargement is not a multiple of 256? We recommend multiplying 1024*1024 by 1.5 to 1536x1536.

Yes, although this seems to show the original result in the Forge gui (and I don't now see the checkerboard version either?), it does indeed save a post-ADetailer transparent copy to the Output directory with the suffix -rebuild. Many thanks new-sanaku!
